src/scry/publish_diagnostic.cr

```diff
@@ -6,7 +6,8 @@ require "./workspace"
 
 module Scry
   struct PublishDiagnostic
-    METHOD = "textDocument/publishDiagnostics"
+    METHOD                = "textDocument/publishDiagnostics"
+    FILES_WITH_DIAGNOSTIC = [] of String
```

Comment:
Why it that a constant and not an ivar?

Reply:
@bew Oh, I just realize we don't need that, we don't need even a var, this is used just on one place, let me fix that :+1:

Reply:
Done!

Reply:
Huh, I was talking about `FILES_WITH_DIAGNOSTIC` :joy:

Reply:
See https://github.com/crystal-lang-tools/scry/pull/95#discussion_r183521067